eo-runtime/src/main/java/EOorg/EOeolang/EOrust.java

```diff
@@ -128,15 +128,29 @@ public EOrust(final Phi sigma) {
                             "EOrust.natives.%s",
                             name
                         )
-                    ).getDeclaredMethod(name, null);
+                    ).getDeclaredMethod(name, new Class[]{EOrust.class});
                     return new Data.ToPhi(
-                        Long.valueOf((int) method.invoke(null))
+                        Long.valueOf((int) method.invoke(null, this))
                     );
                 }
             )
         );
     }
 
+    /**
+     * Finds vertex of eo object by its location.
+     * @param name Relative location of the object to find.
+     * @return Vertex of the object to find.
+     * @todo #2237:45min Implement finding by location.
+     *  Name argument is something like "^.^.some-obj".
+     *  This string must be splitted by '.' and then for
+     *  every part it is necessary to call this.attr().get()
+     * @checkstyle NonStaticMethodCheck (4 lines)
+     */
+    public int find(final String name) {
```

Comment:
@Graur this method will use `this` in its implementation. Now it cannot be static because it is called from EOEnv in rust via `call_mathod`. There is a function `call_static_method` but in rust we does not have EOrust class pointer so the rust code would be more complicated if the method was static.

Reply:
@levBagryansky Ok than, thanks!